docs/articles_en/documentation/openvino-ir-format/operation-sets/operation-specs/movement/strided-slice-1.rst

```diff
@@ -14,6 +14,69 @@ StridedSlice
+* If :math:`begin[i] == end[i]`, only a single element from the corresponding dimension is added to the output. The corresponding output dimension is then equal to 1 (in other words, the dimension is kept).
+* At each step, the :math:`slicing\_index` is incremented by the value of :math:`stride[i]`. As long as the :math:`slicing\_index < end[i]`, the element corresponding to the :math:`slicing\_index` is added to the output.
+* Whenever :math:`slicing\_index >= end[i]`, the slicing stops, and the corresponding element is not added to the output.
+
+Notice that the basic slicing operation assumes N = M (that is, i-th slicing step corresponds to i-th dimension), as no masks are used.
+
+.. note:: Indexing in Reverse (Slicing in Reverse)
+
+   If :math:`stride[i] < 0`, the indexing will happen in reverse. At each step, the value of :math:`stride[i]` will be subtracted from the :math:`slicing\_index`. As long as the :math:`slicing\_index > end[i]`, the corresponding element is added to the output. Whenever :math:`slicing\_index <= end[i]`, the slicing stops.
+
+.. note:: Value Out-of-Bounds (Silent Clamping)
+
+   If a value in begin or end is out of bounds for the corresponding dimension, it is silently clamped. In other words:
+
+   * If :math:`begin[i] >= size(dim)`, then :math:`begin[i] = size(dim)`. If :math:`begin[i] < 0` (after Negative Values Adjusting), then :math:`begin[i] = 0`.
+   * If :math:`end[i] >= size(dim)`, then :math:`end[i] = size(dim)`. If :math:`end[i] < 0` (after Negative Values Adjusting), then :math:`end[i] = 0`.
+
+   If slicing in reverse, the clamping behavior changes to the following:
+
+   * If :math:`begin[i] >= size(dim)`, then :math:`begin[i] = size(dim) - 1`. If :math:`begin[i] < 0` (after Negative Values Adjusting), then :math:`begin[i] = 0`.
+   * If :math:`end[i] >= size(dim)`, then :math:`end[i] = size(dim)`. If :math:`end[i] < 0` (after Negative Values Adjusting), then :math:`end[i] = -1`.
+
+The operation accepts multiple bitmasks in the form of integer arrays to modify the above behavior. **If the length of the bitmask is less than the length of the corresponding input, it is assumed that the bitmask is extended (padded at the end) with zeros. If the length of the bitmask is greater than necessary, the remaining values are ignored.**
+
+During the i-th slicing step:
+
+* If the :math:`begin\_mask[i]` is set to one, the value of :math:`begin[i]` is set to 0 (size(dim) - 1 if slicing in reverse). Equivalent of swapping left handside of Python slicing operation :math:`array[0:10]` with :math:`array[:10]` (slice from the start).
+* If the :math:`end\_mask[i]` is set to one, the value of :math:`end[i]` is set to size(dim) (0 if slicing in reverse - note that this does not allow slicing inclusively with the first value). Equivalent of swapping right handside of Python slicing operation :math:`array[0:10] (assume len(array) = 10)` with :math:`array[0:]` (slice till the end, inclusive).
```

Comment:
It seem that docs have problem formatting spaces in `:math:` - `assume len` in rendered as `assumelen`. Same in line 23 (`ellipsis_maskinputs`), 63 (`onelessdimension`) and likely more.

Reply:
I tested the :code: block, but I finally decided to keep :math: but just extract the words to keep the professional math-like style